.github/workflows/pr.yml

```diff
@@ -10,17 +10,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
       - uses: actions/setup-node@v3
         with:
           node-version: 18
-          cache: yarn
+          cache: 'pnpm'
       - name: 'Create env file'
         run: |
           echo DEPLOY_ENV=dev > .env
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: pnpm install --frozen-lockfile
       - name: Build website
-        run: yarn build
+        run: pnpm build
```

Comment:
I would consider making all the `pnpm` commands that run `package.json` scripts `pnpm run ...`

without `run`, sometimes pnpm commands can be flakey or have unexpected side-effects.. like not accepting additional flags, arguments, etc

Reply:
Interesting! I was following a dev.to guide that suggested the opposite 😅

Will change